src/tests/functional/plugin/shared/include/behavior/ov_plugin/query_model.hpp

```diff
@@ -39,6 +39,13 @@ TEST_P(OVClassModelTestP, QueryModelWithKSO) {
     }
 }
 
+TEST_P(OVClassModelTestP, ImportModelWithNullContextThrows) {
+    ov::Core ie = createCoreWithTemplate();
+    ov::RemoteContext context;
+    std::istringstream stream("None");
+    ASSERT_THROW(ie.import_model(stream, context, {}), ov::Exception);
```

Comment:
1. why do we have `import_model` test in `query_model.hpp` file?
2. this test SEGFAULTs on NVIDIA https://openvino-ci.toolbox.iotg.sclab.intel.com/blue/organizations/jenkins/private-ci%2Fie%2Fie-tests-linux-ubuntu20-gpu_nvidia/detail/ie-tests-linux-ubuntu20-gpu_nvidia/343/pipeline/

Reply:
@olpipi please a look ASAP